src/Features/OnboardingSetUpShipping.php

```diff
@@ -63,8 +63,13 @@ public static function on_onboarding_profile_completed() {
 			return;
 		}
 
-		$user_token        = \Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
-		$jetpack_connected = isset( $user_token->external_user_id );
+		if ( defined( 'JETPACK_MASTER_USER' ) ) {
+			$user_token = \Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
+			$jetpack_connected = isset( $user_token->external_user_id );
+		} else {
+			$jetpack_connected = apply_filters( 'woocommerce_admin_is_jetpack_connected', false );
```

Comment:
Can you add a comment block for this filter?

Reply:
Done on 52a2357